From 9b5ba7c9ca5ee4714cdf6f79a55f04578db37719 Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Mon, 29 Aug 2022 15:25:55 +0200
Subject: [PATCH 20/45] printk: Remove bogus comment vs. boot consoles

The comment about unregistering boot consoles is just not matching the
reality. Remove it.

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 kernel/printk/printk.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/kernel/printk/printk.c b/kernel/printk/printk.c
index df8c83d2c9b1..9728e8ef62cd 100644
--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -3208,9 +3208,6 @@ void register_console(struct console *newcon)
 	if (bootcon_enabled &&
 	    ((newcon->flags & (CON_CONSDEV | CON_BOOT)) == CON_CONSDEV) &&
 	    !keep_bootcon) {
-		/* We need to iterate through all boot consoles, to make
-		 * sure we print everything out, before we unregister them.
-		 */
 		for_each_console(con)
 			if (con->flags & CON_BOOT)
 				unregister_console(con);
-- 
2.37.3

